packages/vaadin-list-mixin/vaadin-list-mixin.js

```diff
@@ -121,7 +121,7 @@ export const ListMixin = (superClass) =>
      * @return {Element}
      */
     get focused() {
-      return this.getRootNode().activeElement;
+      return (this.items || []).find((item) => item.focused);
```

Comment:
Consider using this helper to check for actual `activeElement`:

https://github.com/vaadin/web-components/blob/57406c5f84b658f7934fc835f95cad4ae63e5dcc/packages/component-base/src/focus-utils.js#L206-L208

I guess we could use it like this:

```js
      return (this.items || []).find(isElementFocused);
 ```

Reply:
Done